platform/linux-generic/odp_packet_io.c

```diff
@@ -638,8 +638,20 @@ static odp_buffer_hdr_t *pktin_dequeue(queue_t q_int)
 	if (pkts <= 0)
 		return NULL;
 
-	if (pkts > 1)
-		queue_fn->enq_multi(q_int, &hdr_tbl[1], pkts - 1);
+	if (pkts > 1) {
+		int num_enq;
+		int num = pkts - 1;
+
+		num_enq = queue_fn->enq_multi(q_int, &hdr_tbl[1], num);
+
+		if (odp_unlikely(num_enq < num)) {
+			if (odp_unlikely(num_enq < 0))
+				num_enq = 0;
+
+			buffer_free_multi(&hdr_tbl[num_enq + 1], num - num_enq);
```

Comment:
Worth an `ODP_DBG()` here? At minimum I'd think we'd want to capture some sort of statistic for these drops.

Same comment for the rest of the similar drops in this commit.

Reply:
ODP_DBG() added in v2